### ISSUE TITLE
Add requirements.txt to project and readme.md

### ISSUE THREAD

**Issue by Andygmb** [Andygmb/twitchy_the_bot#20]

**Comment by Andygmb** [comment]
In setup.py, perhaps it should try importing modules and if they aren't available, inform the user *then* raise the traceback? 

eg:

    try:
        from PIL import Image
    except:
        print "You need PIL installed, see http://pillow.readthedocs.org/porting-pil-to-pillow.html for installation instructions."
        raise

**Comment by andrewalexander** [comment]
Yeah that seems about right. Though rather than print, why not just pass that text in as the exception text like `raise("You need PIL installed, see http://pillow.readthedocs.org/porting-pil-to-pillow.html for installation instructions.")`? I think it is the more pythonic way to handle it, no?

**Comment by Andygmb** [comment]
My thinking with using print over raising an exception is that a lot of the people who would use this bot aren't super familiar with python and while the tracebacks from exceptions are pretty easy to understand anyhow, a lot of people see a traceback and their mind immediately shuts down. 

I've had people report issues with the bot where their answer is in the traceback they paste, but they just don't seem to read it. I would hope the "prettier" print statements would help those people understand what's going wrong. 

I include the exception after the print as well though, so maybe I should just switch to using Raise Exception("Error") instead - i'm open to changing it, but my main concern with all these things is to make it as easy as possible for someone who barely knows python to use the bot and not become confused.

**Comment by andrewalexander** [comment]
Ah... fair enough. I think raise Exception("Error") can be a fair compromise - have a verbose and friendly message saying in plain english what went wrong and then have the technical details above for people who know what's going on. But ultimately it's up to you :smile:
